corehq/apps/analytics/tasks.py

```diff
@@ -38,6 +38,19 @@
 HUBSPOT_EXISTING_USER_INVITE_FORM = "7533717e-3095-4072-85ff-96b139bcb147"
 HUBSPOT_COOKIE = 'hubspotutk'
 
+ANALYTICS_RETRIES = 3
+ANALYTICS_SLEEP = 5
+
+def _persistent_analytics_post(func, retries=ANALYTICS_RETRIES, sleep=ANALYTICS_SLEEP):
+    for i in range(retries):
+        try:
+            return func()
+        except (requests.exceptions.HTTPError, requests.exceptions.BaseHTTPError):
```

Comment:
Should we retry only on certain error-codes and not all? (for e.g. we don't need to retry if the error is related to our post data being invalid. Here is such [failure](http://manage.dimagi.com/default.asp?190179))